modules/eks/iam-workers.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_iam_role" "demo-node" {
+  name               = "terraform-eks-demo-node"
+  assume_role_policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+POLICY
+
+}
```

Comment:
newline is missing here and in the rest of the file as well

Reply:
fixed